www/js/charts/histogramChartDirective.js

```diff
@@ -140,8 +140,16 @@ export default ($log) => {
             // Frequency for how tall each bar is on left Y-Axis
             let frequency = data.map((d) => { return d.frequency; }),
                 maxFrequency = max(frequency),
-                minFrequency = min(frequency),
-                barHeight = barContainerHeight / maxFrequency;
+                minFrequency = min(frequency);
+
+            let domainMax = max(frequency),
```

Comment:
instead of using max(frequency) for the domain maximum, lets set the maximum to a value divisible by the number of ticks we want. In this case, 5.